modules/odc-importing-codebase-from-git-to-create-application.adoc

```diff
@@ -48,7 +48,7 @@ The resource name must be unique in a namespace. Modify the resource name if you
 +
 [NOTE]
 ====
-The *Serverless Deployment* option is displayed in the *Import from git* form only if the {ServerlessOperatorName} is installed in your cluster. For further details, refer to the {ServerlessProductName} documentation.
+You can set the default resource preference for Import by browsing the User Preferences page and clicking *Applications* -> *Resource type* field. The *Serverless Deployment* option is displayed in the *Import from git* form only if the {ServerlessOperatorName} is installed in your cluster. For further details, refer to the {ServerlessProductName} documentation.
```

Comment:
Vale complains about the passive voice here ("is displayed", "is installed"), but personally I think it reads better as it is. 

You could say "The *Import from Git* form displays the *Serverless Deployment* option only ..." but I don't think it really improves it.

The only place I would change the passive voice is in the sentence:

"The changes made to the user preferences..." -> "The changes you make to the user preferences.."

but even that is a weak suggestion, and not required.

Reply:
As per your above comment, I am not modifying any content here, as it is not needed. Thank you!